CRM/Member/Import/Parser/Membership.php

```diff
@@ -371,20 +368,9 @@ public function import($onDuplicate, &$values) {
               CRM_Price_BAO_LineItem::getLineItemArray($formatted, NULL, 'membership', $formatted['membership_type_id']);
             }
 
-            // @todo stop passing $ids array (and put details in $formatted if required)
-            $ids = [
-              'membership' => $formatValues['membership_id'],
-              'userId' => $session->get('userID'),
-            ];
-            $newMembership = CRM_Member_BAO_Membership::create($formatted, $ids, TRUE);
-            if (civicrm_error($newMembership)) {
```

Comment:
exceptions are caught further down in the function as the api is used elsewhere in this class